website/src/docs/fileinput.md

```diff
@@ -23,6 +23,40 @@ uppy.use(FileInput, {
 
 The `@uppy/xhr-upload` example uses `@uppy/file-input` with the [`pretty`](#pretty-true) option enabled.
 
+## Custom file input
```

Comment:
In the S3 plugins, we have additional sections below the `Options` one:
![image](https://user-images.githubusercontent.com/1006268/62628981-9c178f00-b92c-11e9-8fa8-bf5d81f62193.png)
for consistency, it might be best to move this to the end as well